explorer/templates.go

```diff
@@ -151,6 +151,42 @@ func float64Formatting(v float64, numPlaces int, useCommas bool, boldNumPlaces .
 	return []string{integer, dec[:places], dec[places:], trailingZeros}
 }
 
+func amountAsDecimalPartsTrimmed(v, numPlaces int64, useCommas bool) []string {
+	amt := strconv.FormatInt(v, 10)
+	intpart := v / 1e8
+	decpart := v % 1e8
+
+	if len(amt) <= 8 {
+		right := strings.TrimRight(amt, "0")
+		left := ""
+		tail := strings.Repeat("0", 8-len(amt))
+		if (8 - len(amt)) > 2 {
+			tail = "00"
```

Comment:
What is this about?  If the idea is to limit to 2 zeros max, that needs to be documented.

Reply:
Documented, it was so it would not leave value as just _1._ with nothing trailing